[0.25][3.1.190618.2-rc1] - Asset BTS - white sreen

**Describe the bug**
A white screen appears when opening the 'Asset BTS' page with an interface language other than English. Russian, Turkish, Spanish, etc.
Related with #2789, but there was a problem only with the Russian locale 'ru' help files (perhaps the problem with other languages remained unnoticed).

**To Reproduce**
Steps to reproduce the behavior:
1. Change the language
2. Go to [staging.bitshares.org/#/asset/BTS](https://staging.bitshares.org/#/asset/BTS)
3. See white screen

**Expected behavior**
Don`t see a white screen.

**Screenshots**
![BTS - white screen](https://user-images.githubusercontent.com/17462020/60665683-d6919600-9e6d-11e9-870e-22faf7d7c843.jpg)

![MxKuJFwYex](https://user-images.githubusercontent.com/17462020/60665708-ec06c000-9e6d-11e9-92e7-c1ae3e0245ff.gif)


**Desktop (please complete the following information):**
 - Windows 7
 - Chrome 71

**Additional context**
```
app.07d6a517c7471249bde0.js:1 missing path 'assets/BTS' for locale 'es' help files, rolling back to alt_path 'assets/Asset'
n @ app.07d6a517c7471249bde0.js:1
console.warn @ app.07d6a517c7471249bde0.js:1
value @ app.07d6a517c7471249bde0.js:1
Bi @ vendor.0c03c0a87759ca1ced2a.js:148
Ni @ vendor.0c03c0a87759ca1ced2a.js:148
da @ vendor.0c03c0a87759ca1ced2a.js:148
ha @ vendor.0c03c0a87759ca1ced2a.js:148
Ga @ vendor.0c03c0a87759ca1ced2a.js:148
qa @ vendor.0c03c0a87759ca1ced2a.js:148
Ka @ vendor.0c03c0a87759ca1ced2a.js:148
Ua @ vendor.0c03c0a87759ca1ced2a.js:148
ga @ vendor.0c03c0a87759ca1ced2a.js:148
enqueueSetState @ vendor.0c03c0a87759ca1ced2a.js:148
b.setState @ vendor.0c03c0a87759ca1ced2a.js:140
t @ vendor.0c03c0a87759ca1ced2a.js:32
(anonymous) @ vendor.0c03c0a87759ca1ced2a.js:32
app.07d6a517c7471249bde0.js:1 missing alt_path 'assets/Asset' for locale 'es' help files, rolling back to 'en'
n @ app.07d6a517c7471249bde0.js:1
console.warn @ app.07d6a517c7471249bde0.js:1
value @ app.07d6a517c7471249bde0.js:1
Bi @ vendor.0c03c0a87759ca1ced2a.js:148
Ni @ vendor.0c03c0a87759ca1ced2a.js:148
da @ vendor.0c03c0a87759ca1ced2a.js:148
ha @ vendor.0c03c0a87759ca1ced2a.js:148
Ga @ vendor.0c03c0a87759ca1ced2a.js:148
qa @ vendor.0c03c0a87759ca1ced2a.js:148
Ka @ vendor.0c03c0a87759ca1ced2a.js:148
Ua @ vendor.0c03c0a87759ca1ced2a.js:148
ga @ vendor.0c03c0a87759ca1ced2a.js:148
enqueueSetState @ vendor.0c03c0a87759ca1ced2a.js:148
b.setState @ vendor.0c03c0a87759ca1ced2a.js:140
t @ vendor.0c03c0a87759ca1ced2a.js:32
(anonymous) @ vendor.0c03c0a87759ca1ced2a.js:32
app.07d6a517c7471249bde0.js:1 Error: Minified React error #185; visit https://reactjs.org/docs/error-decoder.html?invariant=185 for the full message or use the non-minified dev environment for full errors and additional helpful warnings. 
    at e.exports (vendor.0c03c0a87759ca1ced2a.js:148)
    at p (vendor.0c03c0a87759ca1ced2a.js:148)
    at ga (vendor.0c03c0a87759ca1ced2a.js:148)
    at Object.enqueueSetState (vendor.0c03c0a87759ca1ced2a.js:148)
    at t.b.setState (vendor.0c03c0a87759ca1ced2a.js:140)
    at t.value (app.07d6a517c7471249bde0.js:1)
    at n (app.07d6a517c7471249bde0.js:1)
    at console.error (app.07d6a517c7471249bde0.js:1)
    at Fi (vendor.0c03c0a87759ca1ced2a.js:148)
    at t.Wi.r.function.r.componentDidCatch.n.callback (vendor.0c03c0a87759ca1ced2a.js:148)
n @ app.07d6a517c7471249bde0.js:1
console.error @ app.07d6a517c7471249bde0.js:1
Fi @ vendor.0c03c0a87759ca1ced2a.js:148
Wi.r.function.r.componentDidCatch.n.callback @ vendor.0c03c0a87759ca1ced2a.js:148
Qo @ vendor.0c03c0a87759ca1ced2a.js:148
Ko @ vendor.0c03c0a87759ca1ced2a.js:148
Xa @ vendor.0c03c0a87759ca1ced2a.js:148
Ga @ vendor.0c03c0a87759ca1ced2a.js:148
qa @ vendor.0c03c0a87759ca1ced2a.js:148
Ka @ vendor.0c03c0a87759ca1ced2a.js:148
Ua @ vendor.0c03c0a87759ca1ced2a.js:148
ga @ vendor.0c03c0a87759ca1ced2a.js:148
enqueueSetState @ vendor.0c03c0a87759ca1ced2a.js:148
b.setState @ vendor.0c03c0a87759ca1ced2a.js:140
t @ vendor.0c03c0a87759ca1ced2a.js:32
(anonymous) @ vendor.0c03c0a87759ca1ced2a.js:32
app.07d6a517c7471249bde0.js:1 BindToChainState(t) Error: Minified React error #185; visit https://reactjs.org/docs/error-decoder.html?invariant=185 for the full message or use the non-minified dev environment for full errors and additional helpful warnings. 
    at e.exports (vendor.0c03c0a87759ca1ced2a.js:148)
    at p (vendor.0c03c0a87759ca1ced2a.js:148)
    at ga (vendor.0c03c0a87759ca1ced2a.js:148)
    at Object.enqueueSetState (vendor.0c03c0a87759ca1ced2a.js:148)
    at u.b.setState (vendor.0c03c0a87759ca1ced2a.js:140)
    at u.value (app.07d6a517c7471249bde0.js:1)
    at u.value (app.07d6a517c7471249bde0.js:1)
    at u.value (app.07d6a517c7471249bde0.js:1)
    at li (vendor.0c03c0a87759ca1ced2a.js:148)
    at Ni (vendor.0c03c0a87759ca1ced2a.js:148) undefined
n @ app.07d6a517c7471249bde0.js:1
console.error @ app.07d6a517c7471249bde0.js:1
value @ app.07d6a517c7471249bde0.js:1
value @ app.07d6a517c7471249bde0.js:1
value @ app.07d6a517c7471249bde0.js:1
li @ vendor.0c03c0a87759ca1ced2a.js:148
Ni @ vendor.0c03c0a87759ca1ced2a.js:148
da @ vendor.0c03c0a87759ca1ced2a.js:148
ha @ vendor.0c03c0a87759ca1ced2a.js:148
Ga @ vendor.0c03c0a87759ca1ced2a.js:148
qa @ vendor.0c03c0a87759ca1ced2a.js:148
Ka @ vendor.0c03c0a87759ca1ced2a.js:148
Ua @ vendor.0c03c0a87759ca1ced2a.js:148
ga @ vendor.0c03c0a87759ca1ced2a.js:148
enqueueSetState @ vendor.0c03c0a87759ca1ced2a.js:148
b.setState @ vendor.0c03c0a87759ca1ced2a.js:140
t @ vendor.0c03c0a87759ca1ced2a.js:32
(anonymous) @ vendor.0c03c0a87759ca1ced2a.js:32
```

--- a/app/components/Blockchain/Asset.jsx
+++ b/app/components/Blockchain/Asset.jsx
@@ -2066,12 +2066,15 @@ class Asset extends React.Component {
                                     {this.renderFeesClaiming(asset)}
                                     {this.renderAssetOwnerUpdate(asset)}
                                     {"bitasset" in asset &&
-                                    !asset.bitasset.is_prediction_market
-                                        ? this.renderFeedPublish(asset)
-                                        : null}
+                                        !asset.bitasset.is_prediction_market &&
+                                        this.renderFeedPublish(asset)}
                                     {this.state.collateralBids.length > 0 &&
                                         this.renderCollateralBid(asset)}
-                                    {this.renderAssetResolvePrediction(asset)}
+                                    {"bitasset" in asset &&
+                                        !asset.bitasset.is_prediction_market &&
+                                        this.renderAssetResolvePrediction(
+                                            asset
+                                        )}
                                 </div>
                             </Tab>
                         </Tabs>

--- a/app/components/Blockchain/Asset.jsx
+++ b/app/components/Blockchain/Asset.jsx
@@ -2071,7 +2071,7 @@ class Asset extends React.Component {
                                     {this.state.collateralBids.length > 0 &&
                                         this.renderCollateralBid(asset)}
                                     {"bitasset" in asset &&
-                                        !asset.bitasset.is_prediction_market &&
+                                        asset.bitasset.is_prediction_market &&
                                         this.renderAssetResolvePrediction(
                                             asset
                                         )}

--- a/app/components/Blockchain/AssetResolvePrediction.jsx
+++ b/app/components/Blockchain/AssetResolvePrediction.jsx
@@ -2,7 +2,6 @@ import React from "react";
 import Translate from "react-translate-component";
 import BindToChainState from "../Utility/BindToChainState";
 import ChainTypes from "../Utility/ChainTypes";
-import classnames from "classnames";
 import AssetActions from "actions/AssetActions";
 import counterpart from "counterpart";
 import {Radio, Tooltip, Button, Form} from "bitshares-ui-style-guide";

